### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by davodesign84**
Fix uppy on collection form
show full date on hover for article cards
fix date for homepage article cards
filter test accounts from discover pages

### CODE REVIEW

File: packages/kauri-web/components/containers/Discover/Articles/List.ts [hunk 1]
```diff
@@ -35,6 +35,11 @@ export default compose(
       variables: {
         filter: {
           mustNotContainTag: ["ethdenver-2019-submission"],
+          mustNotIncludeUserId: [
+            "b282635ffc0ea4d6984f6b50e9dab90de1d03ce2",
```
Comment (rej156) [inline packages/kauri-web/components/containers/Discover/Articles/List.ts]: Move to config

File: packages/kauri-components/components/Card/ArticleCard.tsx [hunk 2]
```diff
@@ -241,6 +241,34 @@ interface ICardContentProps {
...
+      status: props.status,
+    }
+  }
+
+  public render() {
+    return (
+      <div onMouseEnter={() => this.setState({ hovered: true})} onMouseLeave={() => this.setState({ hovered: false})}>
+        <Label>
+          {this.state.hovered ?
+            (this.props.status === "DRAFT" ? "Drafted " : "Posted ") + moment(this.props.date).format('DD MMM YYYY hh:mm') :
```
Comment (rej156) [inline packages/kauri-components/components/Card/ArticleCard.tsx]: If users don't complain, don't do it

Comment (davodesign84) [reply packages/kauri-components/components/Card/ArticleCard.tsx]: That's been requested by Kendall and Nelson

File: packages/kauri-web/components/containers/Discover/Articles/List.ts (hunk 1 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/Discover/Articles/List.ts]: fair point :D

File: packages/kauri-components/components/Card/ArticleCard.tsx (hunk 2 above)
Comment (rej156) [reply packages/kauri-components/components/Card/ArticleCard.tsx]: Same thingg

**Pull request merged**